src/chainer.js

```diff
@@ -8,11 +10,13 @@ function chainer(functions) {
     let result = x;
```

Comment:
The variable 'x' is not defined anywhere in the function. Please make sure to define or pass it as a parameter to the function.